Library/Homebrew/style.rb

```diff
@@ -321,22 +321,26 @@ def self.github_workflow_files
     end
 
     def self.rubocop
+      require "formula"
       ensure_formula_installed!("rubocop", latest: true,
                                            reason: "Ruby style checks").opt_bin/"rubocop"
     end
```

Comment:
This one is odd, because I don't think we have a `rubocop` formula?

Reply:
You might find this is never called.

Reply:
Cool; let's try removing it.